builders/tanssi-network/token.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Utility Token
+description: Learn about the two versions of the Tanssi token - the native Substrate token and the ERC-20 representation on Ethereum, their utilities, and use cases.
+icon: octicons-ruby-24
+---
+
+# Utility Token {: #utility-token }
+
+## Introduction {: #introduction }
+
+The Tanssi network token is the utility token that powers the Tanssi protocol. Considering [Tanssi's architecture](/learn/tanssi/overview/#tanssi-architecture){target=\_blank}, the token exists in two distinct yet interconnected representations: native substrate and Ethereum ERC20. The two versions can be bridged between each other through the [Tanssi-Ethereum bridge](/learn/tanssi/tanssi-ethereum-bridge/){target=\_blank}.
+
+In this guide, the token's utility and the differences between its two representations are covered, which is crucial for network operators, stakers, appchain managers, and general users who want to participate in the Tanssi ecosystem.
+
+## Token Utility {: #token-utility }
+
+Tanssi is a decentralized infrastructure protocol that makes deploying appchains with custom logic easy. It allows developers to focus on the use case instead of diverting time and energy to manage the [numerous components required](/learn/tanssi/overview/#what-tanssi-provides){target=\_blank} for a network to run smoothly.
+
+Running a healthy decentralized protocol requires aligning incentives and coordinating among several ecosystem actors, including appchain developers, node operators, sequencer operators, data availability and RPC providers, as well as general users. The Tanssi token serves as the backbone, providing the economic mechanisms necessary to coordinate, incentivize proper behavior, and secure the entire ecosystem.
```

Comment:
I think we should add governance here, which is crucial for a decentralized protocol etc.